src/page/QuotePage.jsx

```diff
@@ -6,7 +6,8 @@ const QuotePage = () => {
 
   useEffect(() => {
     const category = 'happiness';
-    const apiKey = import.meta.env.VITE_NINJAS_QUOTES_API_KEY;
+    // const apiKey = import.meta.env.VITE_NINJAS_QUOTES_API_KEY;
+    const apiKey = process.env.VITE_NINJAS_QUOTES_API_KEY;
```

Comment:
Is the page still working with this change?

Reply:
yes it is

Reply:
no, but works with the commented out code